src/components/Text/MarkedText/MarkedText.tsx

```diff
@@ -1,15 +1,23 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
+import { Context } from '@textkernel/oneui/constants';
 import { bem } from '../../../utils';
 import { Text } from '../Text';
 import styles from './MarkedText.scss';
 
 const { block, elem } = bem('MarkedText', styles);
 
-export const MarkedText = (props) => {
+interface Props {
```

Comment:
looking at it more, I think some of the props are actually TextProps. Maybe it makes better sense to use `extend TextProps` here, and not list `inline` and `context`

Reply:
You're right!! Fixed.